open_src/influx/meta/data.go

```diff
@@ -2769,6 +2805,24 @@ func (data *Data) checkDDLConflict(e *proto2.MigrateEventInfo) error {
 	return nil
 }
 
+// RegisterQueryIDOffset register the mapping relationship between its host and query id offset for ts-sql
+func (data *Data) RegisterQueryIDOffset(host string) error {
+	if data.QueryIDInit == nil {
+		data.QueryIDInit = make(map[string]uint64)
+	}
+
+	currentAssignedNum := len(data.QueryIDInit)
+	if currentAssignedNum >= MaxAssignableNum {
+		return errno.NewError(errno.QueryIDOverflow)
```

Comment:
`MaxAssignableNum = 1.8446744 * 10^14`
It is impossible to have so many SQLs.
I suggest not adding this `if`